src/chi/components/drawer/drawer.scss

```diff
@@ -180,6 +180,11 @@ $animation-duration: 0.5s;
 
   .-close {
     position: fixed;
+    /* IE11 CSS hack */
```

Comment:
@janibolkvadze hack code is usually placed at the bottom of the scss file. Please change